src/app/_components/TopBar/TopBar.js

```diff
@@ -20,7 +20,16 @@ export default function TopBar({ title, details, slug }) {
             <p className={styles.header}>{title}</p>
           </>
         )}
+
+        <>
+        <Link href="/app/settings">
+          <div className={styles.settingsContainer}>
+            {/* <FaCog size={22} color={"white"} /> */}
```

Comment:
This is a settings gear icon to make it look nicer. If I add this, the app crashes so I commented it out. ANyone know how to fix this?

Reply:
I moved the gear to the bottom and added the icon